common/src/service/ExceptionHandlingHelper.cs

```diff
@@ -17,24 +17,23 @@ class ExceptionHandlingHelper
         {
             var mappings = new Dictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>>();
 
```

Comment:
Previously, this resulted in setting wrong error messages for DeviceNotFoundException as the message was being passed in as a deviceId. I have updated all methods to explicitly use the correct method and parameters so that if we add any overloads, they accidentally don't end up matching something incorrect.